src/pages/CharacterDetailPage.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import CharacterDetail from '../components/character-detail/CharacterDetail';
+
+const CharacterDetailPage = () => {
+  <CharacterDetail />;
```

Comment:
When writing a functional component you need to make sure to return the jsx. You can do this with an implicit return using parentheses but if you use curlies after the arrow you need to explicitly return the jsx.